src/tests/JIT/Regression/JitBlue/Runtime_76194/Runtime_76194.csproj

```diff
@@ -3,8 +3,10 @@
     <OutputType>Exe</OutputType>
     <Optimize>True</Optimize>
     <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
+    <CLRTestTargetUnsupported Condition="'$(TargetOS)' == 'OSX'">true</CLRTestTargetUnsupported>
```

Comment:
What prevents the test from working on macOS?

Reply:
hm.. not sure actually, I think the previous version with DllImport just didn't work on macOS